tools/MUXPGODatabase/NuSpecs/MUXPGODatabase.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>MUXPGODatabase</id>
+    <version>0.0.1</version>
+    <title>MUXPGODatabase</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>profile guided optimization database obtained after running training scenarios.</description>
+    <releaseNotes>Initial release.</releaseNotes>
+    <copyright>Copyright 2017</copyright>
+  </metadata>
+  <files>
+    <file target="build\MUXPGO.targets" src="build\MUXPGO.targets"/>
```

Comment:
Convention is that the .targets file is named the same as the nupkg (i.e. MUXPGODatabase.targets)